aws/data_source_aws_lb.go

```diff
@@ -1,11 +1,13 @@
 package aws
 
 import (
+	"bytes"
 	"fmt"
 	"log"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/hashcode"
```

Comment:
This import was very recently moved as part of our Terraform Plugin SDK v1 removal: #14593

It now lives at:

```go
"github.com/terraform-providers/terraform-provider-aws/aws/internal/hashcode"
```

That being said, adding `Set` functions to data sources should generally not be necessary and especially after this fix in SDK version 2: https://github.com/hashicorp/terraform-plugin-sdk/pull/197

If this is rebased against the current commit on the main branch, which is all up to date with SDK v2, the `Set` function can hopefully be removed.